models/Product.php

```diff
@@ -609,8 +609,8 @@ public function filterFields($fields, $context = null)
      */
     protected function hideField($fields, string $field)
     {
-        if (property_exists($fields, $field)) {
-            $fields->$field->hidden = true;
+        if (array_key_exists($field, $fields->config)) {
+            $fields->config[$field]->hidden = true;
         }
```

Comment:
I am now getting the error "Undefined property: stdClass::$config"

/plugins/offline/mall/models/Product.php line 612

Reply:
Looks like an incompatibility. October 3 uses the `ElementHolder` class to pass around these field values.

Reply:
A workaround was added in fda9ecf, thank you @mjauvin!